Fix_Path
"mlflow/utils/uri.py:[424,451]:validate_path_is_safe"
"tests/server/test_handlers.py:[110,111]:enable_serve_artifacts"
"tests/server/test_handlers.py:[799,804]:test_delete_artifact_mlflow_artifacts_throws_for_malicious_path"
"tests/store/artifact/test_http_artifact_repo.py:[250,263]:test_list_artifacts_malicious_path"
"tests/tracking/test_rest_tracking.py:[516,545]:test_path_validation"
"tests/tracking/test_rest_tracking.py:[522,527]:test_path_validation.assert_response"
"tests/utils/test_uri.py:[792,794]:test_validate_path_is_safe_bad"
"tests/utils/test_uri.py:[752,753]:test_validate_path_is_safe_good"
"tests/utils/test_uri.py:[888,890]:test_validate_path_is_safe_windows_bad"
"tests/utils/test_uri.py:[764,765]:test_validate_path_is_safe_windows_good"